src/widgets/window.js

```diff
@@ -258,7 +259,7 @@ var FlatsealWindow = GObject.registerClass({
             showPermissionsButton = false;
         }
 
-        const isFolded = this._headerLeaflet.folded;
+        const isFolded = this._contentLeaflet.folded;
         if (isFolded) {
             showApplicationsButton = true;
             showPermissionsButton = true;
```

Comment:
One regression:

![Peek 2021-01-23 18-27](https://user-images.githubusercontent.com/481418/105614544-e1da4880-5da8-11eb-8a2e-11e95b3457bb.gif)

I had this [geddan-effect](https://knowyourmeme.com/memes/get-down-geddan) issue before, and I workaround it by preventing the condition that triggered it. See ad7f9541ab4e3f86eed49acde0062502d0e8282f

The root cause is that, the permissions header bar width changes _after_ the leaflet threshold has been passed. It changes because some button are hidden, and the action bar at the bottom is shown. So, it gets in this loops when the window bounces in and out the threshold.